lib/google-apis/auth-client.js

```diff
@@ -176,201 +97,68 @@ AuthClient.prototype._generateErrorCreationRequestOptions = function(payload) {
 };
 
 /**
- * Attempts to pull the next request off the queue and execute it. This could be
- * an already previously made request that is being retried or an entirely new
- * request. This function will check for the existence of a request in the queue
- * before attempting to execute that request against the service. If a request
- * is successfully made or has failed the max number of times this function will
- * affect the requestQueue by shifting the first entry off the queue and then
- * attempting to a catalyze the next request by calling the `_queueNextRequest`
- * function.
- * @function _makeRequest
- * @returns {Undefined} - does not return a value
+ * A stubbed function used to substitute as a callback when the user or
+ * application does not provide a valid callback to the request factory.
+ * @function _cbStubNoOp
+ * @param {Error|Null} err - the error the request execution returned, if any
+ * @param {Object} response - the response object
+ * @param {Any} body - the body derived from the response
+ * @returns {Undefined} - does not return anything
  */
-AuthClient.prototype._makeRequest = function() {
-
-  var self = this;
-  var requestPayloadSlot = [];
-  var thisRequestsPayload = {};
-  var thisRequestsCallback = null;
-  var thisRequestsRetryTracker = {};
-  var thisRequestsOptions = {};
-
-  if (!this._areRequestsStillInQueue()) {
-    this._isRequesting = false;
-
-    return;
-  }
-
-  requestPayloadSlot = this._requestQueue[0];
-  thisRequestsPayload = requestPayloadSlot[0];
-  thisRequestsCallback = requestPayloadSlot[1];
-  thisRequestsRetryTracker = requestPayloadSlot[2];
-  thisRequestsOptions =
-      this._generateErrorCreationRequestOptions(thisRequestsPayload);
+RequestHandler.prototype._cbStubNoOp = function (err, response, body) {
```

Comment:
Does this need to be declared or even exported on the class? Just use an inline no-op function where you need.